wallet/wallet.c

```diff
@@ -1086,6 +1141,16 @@ void wallet_channel_save(struct wallet *w, struct channel *chan)
 		sqlite3_bind_null(stmt, 1);
 	sqlite3_bind_int64(stmt, 2, chan->dbid);
 	db_exec_prepared(w->db, stmt);
+
+	/* save announcement sigs only when we ge announcement*/
```

Comment:
`ge` -> `get`

Reply:
Also these fields ought to be initialized to `null` anyway in the DB, so no need to do this additional initialization.